cert-provider/cert-provider.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/sh
+
+ACME_BIN="$(realpath ~/.acme.sh/acme.sh)"
+
+# colour output helpers...
+reset=$(tput -T xterm sgr0)
+red=$(tput -T xterm setaf 1)
+green=$(tput -T xterm setaf 2)
+yellow=$(tput -T xterm setaf 3)
+blue=$(tput -T xterm setaf 4)
+
+function logError() {
+    echo "${red}[Error]${reset} $1"
+}
+
+function logWarn() {
+    echo "${yellow}[Warn]${reset} $1"
+}
+
+function logInfo() {
+    echo "${blue}[Info]${reset} $1"
+}
+
+function logSuccess() {
+    echo "${green}[Success]${reset} $1"
+}
+
+# join domains with -d separator
+join_domains()  { echo "$1" | sed -e "s/,/ -d /g" -e "s/^/-d /g"; }
+
+DO_CERTS="true"
+if [ -z "$ACTIVE" ]; then
+    logError "ACTIVE must be set. Value can be anything."
+    unset DO_CERTS
```

Comment:
The script seems like it requires a few env vars to be set and prints nice logs if needed, but I don't see the script  exiting if any var is missing.

Reply:
It cannot exit otherwise the HAproxy container will fail to resolve it's hostname and cease starting up. It has to stay running, even if it is not going to be used.